assesspy/tests/test_ci.py

```diff
@@ -8,33 +8,33 @@ class TestCI:
     def metric(self, request):
         return request.param
 
-    @pt.fixture(params=[0.80, 0.90, 0.95])
+    @pt.fixture(params=[0.50, 0.20, 0.10, 0.05])
```

Comment:
I'm big dumb and specified the alpha values as `1 - alpha`, thinking "This is the range of the CI I want i.e. 95%." I fixed the values and pulled the test results used in `assessr`.